packages/world/src/World.sol

```diff
@@ -0,0 +1,422 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { console } from "forge-std/console.sol";
+import { Store, IStoreHook } from "store/Store.sol";
+import { StoreCore } from "store/StoreCore.sol";
+import { Schema } from "store/Schema.sol";
+
+import { OwnerTable, tableId as OwnerTableId } from "./tables/OwnerTable.sol";
```

Comment:
more relevant for autogen than world: UX would probably benefit from having `tableId` have a non-generic name at declaration in all tables, so you'd just import `OwnerTableId`

Reply:
agree! This import wasn't actually used, in the current prototype I have the table libs wrapping around the schema libs to remove the need for explicit handling of the table id, but happy to discuss alternative approaches. How were you thinking of handling it with the autogen?

Reply:
Since I can generate the whole thing, the separation of schema/table stops being necessary. If you want 2 same-schema tables, autogen handles it for you, and since config is in typescript, you can use variables if you want to reuse schemas.
Do you see a use-case where schema is useful by itself other than reusability?

Reply:
One minor use case I see is if you wanted to create "generic systems" that can be called with table ids as arguments (simple example, an `IncrementSystem` that doesn't hard code its `CounterTable` but receives it as an argument, so it can be used for different tables that fit the schema

Reply:
Yeah I was thinking how to do something along those lines. How about 2 autogen modes - specific table, and generic schema? That way we should keep the benefits, but reduce the file count/complexity

Reply:
sounds good to me!